Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -17,103 +17,85 @@ import ArcGISToolkit
+                .alert(
+                    isPresented: alertIsPresented,
+                    error: submissionError,
+                    actions: {}
+                )
+                .overlay {
+                    submittingOverlay
                 }
-                .ignoresSafeArea(.keyboard)
                 .sheet(isPresented: featureFormViewIsPresented) {
-                    FeatureFormView(root: featureForm!, isPresented: featureFormViewIsPresented)
-                        .onFormEditingEvent { editingEvent in
-                            if case .savedEdits = editingEvent,
-                               let table = featureForm?.feature.table as? ServiceFeatureTable,
-                               !editedTables.contains(where: { $0 === table }) {
-                                editedTables.append(table)
-                                updateSubmitButtonVisibility()
-                            }
-                        }
-                }
-                .alert("Error", isPresented: alertIsPresented) {
-                } message: {
-                    if let error = alertError {
-                        Text(error)
-                    }
+                    featureFormView
                 }
-                .overlay {
-                    if isApplyingEdits {
-                        HStack(spacing: 5) {
-                            ProgressView()
-                                .progressViewStyle(.circular)
-                            Text("Applying edits")
-                        }
-                        .padding()
-                        .background(.thinMaterial)
-                        .clipShape(.rect(cornerRadius: 10))
-                    } else {
-                        EmptyView()
-                    }
+                .task(id: identifyScreenPoint) {
+                    guard !editsAreBeingApplied,
+                          let identifyScreenPoint else { return }
+                    await makeFeatureForm(point: identifyScreenPoint, map: mapView)
                 }
                 .toolbar {
-                    if submitButtonVisibility != .hidden {
-                        ToolbarItem(placement: .navigationBarTrailing) {
-                            Button("Submit") {
-                                Task {
-                                    await applyEdits()
-                                }
-                            }
-                        }
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        submitButton
                     }
                 }
         }
     }
 }
 
 extension FeatureFormExampleView {
+    /// An error encountered while submitting edits.
+    enum SubmissionError: LocalizedError {
+        case applyEdits(any Error)
+        case other(String)
+        
+        var errorDescription: String? {
+            switch self {
+            case .applyEdits(let error):
+                String(reflecting: error)
```

Comment:
From the [documentation](https://developer.apple.com/documentation/swift/string/init(reflecting:)): "Creates a string with a detailed representation of the given value, **suitable for debugging.**" (emphasis mine). Using this as a user-facing string isn't modeling best practices.

Reply:
The error handling in sample is not a best practice, but since the `errorAlert` is also a modifier that we "hide" from the user, it is less likely to propagate into user's codebase.
(At least for me, when I copy paste from a sample into a different project and see the missing `errorAlert` compile error, I remove the modifier - thus the reflection won't be there)